plugins/game/lib/api/filters/KalturaUserScorePropertiesFilter.php

```diff
@@ -8,7 +8,7 @@
 class KalturaUserScorePropertiesFilter extends KalturaUserScorePropertiesBaseFilter
 {
 	/**
-	 * @var int
+	 * @var string
```

Comment:
1. need to also change the api for update and delete
2. need to test with client to make sure we have backword compatibility with int